src/components/DocumentationTopic/PrimaryContent/DeclarationGroup.vue

```diff
@@ -70,8 +111,35 @@ export default {
       type: String,
       required: false,
     },
+    declListExpanded: {
```

Comment:
Could you change the name to something like "declarationsExpanded" ?
"decl" is a bit difficult to understand

Reply:
I wanted to make it clear that this is for a list of declarations, but naming this `declarationsListExpanded` is a bit too long. I think `decl` is a pretty common abbreviation.

Reply:
sounds good